web/app/templates/dashboard.hbs

```diff
@@ -0,0 +1,37 @@
+<div class="nav-extension"></div>
+
+<div class="container" role="main">
+
+    <table class="dashboard-header">
+        <tr>
+
+            <td class="logo">
+                <img src='assets/images/logo.png'>
+
+            </td>
+            <td>
+                <div class="bubble">
+                    <h3>Hello there, I've been busy!</h3>
+
+                    <p>I looked through <b>150</b> jobs today.<br>
```

Comment:
These values will eventually come from a model. I haven't created the model yet hence have hardcoded it.